test/bigtest/tests/instance-view-page-test.js

```diff
@@ -6,71 +6,221 @@ import InstanceViewPage from '../interactors/instance-view-page';
 import InstanceEditPage from '../interactors/instance-edit-page';
 import InstanceCreatePage from '../interactors/instance-create-page';
 import ItemViewPage from '../interactors/item-view-page';
+import HoldingsViewPage from '../interactors/holdings-view-page';
 
 describe('InstanceViewPage', () => {
-  setupApplication();
+  describe('User has permissions', () => {
+    setupApplication(
+      { permissions: {
```

Comment:
Probably we don't need to explicitly declare permissions, cause user should have all permissions according to the default state of application setup.